### ISSUE TITLE
Error when importing treatment pathway design

### ISSUE THREAD

**Issue by chrisknoll** [OHDSI/WebAPI#645]
### Expected Behavior
Should be able to import a pathway design into a fresh instance of WebAPI.

### Actual Behavior
Import results in a server error:

```
"localizedMessage":"A different object with the same identifier value was already associated with the session : [org.ohdsi.webapi.cohortdefinition.CohortDefinitionDetails#16]","message":"A different object with the same identifier value was already associated with the session : [org.ohdsi.webapi.cohortdefinition.CohortDefinitionDetails#16]","suppressed":[]},"headers":{"id":"fd3b0b18-1080-ff1b-1401-7cfbd5a93821","timestamp":1539192177455}}
```

[importTest.txt](https://github.com/OHDSI/WebAPI/files/2465726/importTest.txt)

See attached file for test case.

**Comment by chrisknoll** [comment]
Additional Info: this importTest does work on the symposium environment (which is hosted on a flavor of PostgreSQL).  So, either this is a problem with the underlying database implementation, or there was some existing data in the database that led to the 'A different object with the same identifier value was already associated...' error.

**Comment by vantonov1** [comment]
@chrisknoll 
1 There is nothing database-special in the code
2 In our env it is possible to import several times, no errors with existing data
3 It looks like a problem with searching cohort by hashcode. BTW, hashcode for cohort definition details was fixed in August, are you using 2.6 master?

**Comment by anthonysena** [comment]
I'm unable to reproduce this issue on the latest code base so I'm closing it out. Thanks!
